draft-ietf-rtcweb-jsep.xml

```diff
@@ -4018,24 +4018,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>
                 <list style="symbols">
 
-                    <t>Remove or reorder codecs (m=)</t>
-                </list>
-            </t>
-
-            <t>The following parameters may be controlled by options
-            passed into createOffer/createAnswer. As an open issue,
-            these changes may also be be performed by manipulating the
-            SDP returned from createOffer/createAnswer, as indicated
-            above, as long as the capabilities of the endpoint are not
-            exceeded (e.g. asking for a resolution greater than what
-            the endpoint can encode):</t>
-
-            <t>
-                <list style="symbols">
-
-                    <t>[[OPEN ISSUE: This is a placeholder for other
-                    modifications, which we may continue adding as use
-                    cases appear.]]</t>
+                  <t>Remove or reorder codecs (m=)</t>
+                  
```

Comment:
My understanding was that we really decided we wanted to remove all such munging (see https://docs.google.com/presentation/d/1pP4LDQQSyVDUTa20R3Zgmb9PXZJPPcXiSzFOYw64dKc/edit#slide=id.g1274a0be3c_0_258), in favor of real APIs.

Having this one exception seems to be of questionable value. I would recommend that we simply say that the description MUST NOT be modified between createOffer and setLocal; then this whole section can be simplified.